.github/workflows/ci.yml

```diff
@@ -29,7 +29,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest]
-        node_version: [18, 20, 22]
+        node_version: [20, 22, 24]
```

Comment:
I think we can keep node 18 tests by using Vite 6 for it like:
https://github.com/sapphi-red/vite-plugin-static-copy/commit/9c7cf2e9831e52c00d9789ee11c8f0db0ea1c330#diff-71cabc4177e41ea8f15a89eb65398fa8187739f68a2762706d84885cd8b2ddc3R68-R69